tests/tests.rs

```diff
@@ -328,11 +349,7 @@ macro_rules! test_header {
         fn $function() {
             let header = PathBuf::from($header);
             let result = create_bindgen_builder(&header).and_then(|builder| {
-                if let Some(builder) = builder {
-                    compare_generated_header(&header, builder)
-                } else {
-                    Ok(())
-                }
+                compare_generated_header(&header, builder, true)
```

Comment:
How much extra time does the roundtrip add to testing? We may want to make this based on an environment variable like `BINDGEN_TEST_ROUNDTRIP`.

We could:
1. Make sure to set the env variable in CI tests
2. Adding extra logging that makes it clear we are doing roundtrip tests

Reply:
Yeah, I thought of this and I was a bit torn... On one hand, I'd like to make running the right set of tests if you're a contributor just `cargo test`. On the other I agree that this is unlikely to catch a lot of errors during development.

For me `cargo test` is still fast with this change, but I have a very fast machine... I think I'd prefer for now not moving it behind an env variable, just under the principle that "if `cargo test` is green, CI should be green too". But I'm happy to reconsider if this regresses stuff on slower machines significantly.

Reply:
I have a very slow machine, but even I would rather have `cargo test` match CI by default, so I personally am fine with this the way it is.

On the other hand, if I learned how to profile (#939), then maybe I would see how much of my life I was losing.

Reply:
We could also make the an opt-out environment variable like `NO_BINDGEN_TEST_ROUNDTRIP`. That way, developers who want to iterate more quickly can disable this test.

Reply:
Yeah, that sounds better. I'll go for `BINDGEN_DISABLE_ROUNDTRIP_TEST` or something of that sort if you don't mind, to keep the `BINDGEN_` prefix.

Reply:
Sure--sounds good.